cogs/embeds.py

```diff
@@ -347,10 +341,7 @@ async def award(self, ctx, amount: int, member: discord.Member):
             await ctx.channel.send(
                 "Minumum amount of credits to award is 1.")
             return
-        dbprof_member = await db.get_member(member.guild.id, member.id)
-        await db.set_member(
-            dbprof_member['guild_id'], dbprof_member['member_id'], 'cash',
-            dbprof_member['cash'] + amount)
+        await db.add_currency(ctx.guild.id, member.id, amount)
         await ctx.channel.send(
             f"{member.mention} got awarded with {amount} credit(s).")
```

Comment:
I'd like to see this string rephrased as `Awarded {amount} credits to {member.mention}`